BUILDING.md

```diff
@@ -174,6 +174,7 @@ Prerequisites:
 * Basic Unix tools required for some tests,
   [Git for Windows](http://git-scm.com/download/win) includes Git Bash
   and tools which can be included in the global `PATH`.
+* NOTE: If the path to your home directory contains a space, the build will likely fail.
```

Comment:
I’d replace `home` with `build` directory, I don’t think the home directory should really have much of an influence.

Also, this is not specific to Windows. ;)